experiments/targeting.go

```diff
@@ -189,7 +189,8 @@ func NewEqualNode(inputNodes map[string]interface{}) (Targeting, error) {
 func (n *EqualNode) Evaluate(inputs map[string]interface{}) bool {
 	candidateValue := inputs[n.fieldName]
 	switch n := candidateValue.(type) {
-	case int, float64:
+	case int, float64, int8, int16, int32, int64, uint, uint8, uint16, uint32,
+		uint64, float32:
```

Comment:
```suggestion
	case int, int8, int16, int32, int64, uint, uint8, uint16, uint32, uint64,
		float32, float64:
```

a little tidier

Reply:
Alternatively, would this work?

```suggestion
	case constraints.Integer, constraints.Float:
```

https://pkg.go.dev/golang.org/x/exp/constraints

Reply:
`constraints` probably won't work but I'm not 100% sure.

if not using `constraints`, please add an extra blank line after this line: https://github.com/reddit/baseplate.go/blob/master/Style.md#blank-line-to-highlight-change-of-logical-level